src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -91,6 +91,7 @@ const GraphDisplay = ({ lensName, setCurNode }: GraphDisplayProps) => {
 
 	const nodeHover = useCallback(
 		(node, ctx) => {
+			console.log("node", node)
```

Comment:
Seems better to make this console.debug

Reply:
Removed console.log - this was just for debugging. Thanks!